doc/source/whatsnew/v3.0.0.rst

```diff
@@ -32,7 +32,7 @@ Other enhancements
 - :func:`read_stata` now returns ``datetime64`` resolutions better matching those natively stored in the stata format (:issue:`55642`)
 - Allow dictionaries to be passed to :meth:`pandas.Series.str.replace` via ``pat`` parameter (:issue:`51748`)
 - Support passing a :class:`Series` input to :func:`json_normalize` that retains the :class:`Series` :class:`Index` (:issue:`51452`)
--
+- Updated :meth:`DataFrame.to_excel` so that the output spreadsheet has no styling. (:issue:`54154`)
```

Comment:
Can you put this under `Other API Changes` and also mention that styling can still be done with `Styler.to_excel`?